build-system/tasks/performance/measure-documents.js

```diff
@@ -156,24 +206,33 @@ async function measureDocument(url, version, {headless}) {
     args: [
       '--allow-file-access-from-files',
       '--enable-blink-features=LayoutInstabilityAPI',
+      '--disable-web-security',
     ],
   });
 
   const page = await browser.newPage();
   await page.setCacheEnabled(false);
+  await page.setRequestInterception(true);
   await setupMeasurement(page);
 
+  const endTimes = [];
+  const startTime = Date.now();
+  page.on('request', interceptedRequest =>
+    handleRequests(interceptedRequest, endTimes)
+  );
+
   try {
     await page.goto(`file:${urlToCachePath(url, version)}`, {
-      waitUntil: 'networkidle0',
+      waitUntil: 'networkidle2',
```

Comment:
I read: networkidle2 - consider navigation to be finished when there are no more than 2 network connections for at least 500 ms.

Compared to `networkidle0`, this will wait less time? Is this expected?

Reply:
It is less wait time, but actually I think that `networkidle0` is what we want, because it will give the most time for analytics events to be sent out.